bin/cli

```diff
@@ -2,7 +2,7 @@
 'use strict'
 
 const program = require('commander')
-const orchestrator = require('../lib/main')
+const orchestrator = require('../src/main')
```

Comment:
This change should be revert.
Hence it will not be deployable.
To test you need to yarn pack the application.
Test are based on the compiled sources.